drivers/soc/aspeed/Makefile

```diff
@@ -10,3 +10,6 @@ obj-$(CONFIG_ASPEED_XDMA)		+= aspeed-xdma.o
 obj-$(CONFIG_ASPEED_UDMA)		+= aspeed-udma.o
 obj-$(CONFIG_AST2600_ESPI)              += ast2600-espi.o
 obj-$(CONFIG_AST2700_ESPI)              += ast2700-espi.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-phy.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-phy3.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-hp.o
```

Comment:
What is hp?

Reply:
Host controller over PCIe, adding the nodes "usb3ahp"  enables xhci over pci